cypress/integration/cookieBanner.js

```diff
@@ -0,0 +1,80 @@
+import {
+  retrieveCookieValue,
+  expectCookieExpiryDateOneYear,
+} from '../support/metaTestHelper';
+import config from '../support/config';
+
+const getPrivacyBanner = () =>
+  cy.contains("We've updated our Privacy and Cookies Policy");
+
+const getCookieBanner = () => cy.contains('Let us know you agree to cookies');
+
+describe('Article Body Tests', () => {
+  // eslint-disable-next-line no-undef
+  beforeEach(() => {
+    cy.visit(`/news/articles/${config.assets.newsThreeSubheadlines}`);
+  });
+
+  it('should have a privacy & cookie banner, which disappears once "accepted" ', () => {
+    getPrivacyBanner().should('be.visible');
+    getCookieBanner().should('not.be.visible');
+
+    retrieveCookieValue('ckns_privacy', '1');
+    retrieveCookieValue('ckns_policy', '000');
```

Comment:
We should call `expectCookieExpiryDateOneYear` on `ckns_privacy` and `ckns_policy` before banner interation